client/src/app/components/forms/inputs/forms/base-command-form.component.ts

```diff
@@ -19,8 +19,10 @@ require("./form.components.scss");
                         <label>Base Command</label>
                         <label class="secondary-label">What command do you want to call from the image</label>
                         
-                        <base-command-input [inputControl]="baseCommandForm.controls['baseCommand']"
-                                        [baseCommand]="baseCommand"></base-command-input>
+                        <expression-input [inputControl]="baseCommandForm.controls['baseCommand']"
+                                            [expression]="baseCommand">
```

Comment:
Because I wanted the Form Control inside the expression-input to be part of the FormGroup on the form, so I would get the validity of the input field.